src/cdk/v2/destinations/algolia/rtWorkflow.yaml

```diff
@@ -28,10 +27,14 @@ steps:
       $.outputs.transform#idx.error.(
         $.handleRtTfSingleEventError(^[idx], .originalError ?? ., {})
       )[]
+
   - name: batchSuccessfulEvents
     description: Batches the successfulEvents
     template: |
-      let batches = $.chunk($.outputs.successfulEvents, $.MAX_BATCH_SIZE);
+      const dontBatchTrueEvents =  $.outputs.successfulEvents{.metadata.dontBatch}[];
+      const dontBatchFalseEvents = $.outputs.successfulEvents{!.metadata.dontBatch}[];
```

Comment:
```suggestion
      const dontBatchEvents =  $.outputs.successfulEvents{.metadata.dontBatch}[];
      const batchEvents = $.outputs.successfulEvents{!.metadata.dontBatch}[];
```

Reply:
going with this implementation to distinguish with flag naming